File: movienotice.webapp/src/app/Models/Movie.ts
export class Movie {
    constructor(
      public id: number = -1,
      public imdbId: string = '',
      public description: string = '',
      public title: string = '',
      public posterUrl: string = '',
      public releaseDate: string = '',
      public backdropPath: string = '',
      public popularity: number = -1
    ) { }
  }

File: MovieNotice.API/Entities/Movie.cs
using MovieNotice.Common.Enums;

namespace MovieNotice.API.Entities
{
    public class Movie
    {   public int Id { get; set; }
        public int TmdbId { get; set; }
        public string ImdbId { get; set; }
        public string Description { get; set; }
        public string Title { get; set; }
        public string PosterUrl { get; set; }
        public DateTime? ReleaseDate { get; set; }
        public string BackdropPath { get; set; }
        public double? Popularity { get; set; }
        public MovieStatus MovieStatus { get; set; }
    }
}
